client/app/services/dashboard.js

```diff
@@ -142,6 +142,8 @@ function transformSingle(dashboard) {
     dashboard.widgets = prepareDashboardWidgets(dashboard.widgets);
   }
   dashboard.publicAccessEnabled = dashboard.public_url !== undefined;
+  // Temporary
+  dashboard.options = { globalParamOrder: [] };
```

Comment:
Just until business logic is added

Reply:
@gabrieldutra brought this back so that you don't need to clone it locally (now it shouldn't break Netlify preview)

Reply:
No need to keep it, I'm already testing locally with the backend chages.